content/basics/use-ably.textile

```diff
@@ -0,0 +1,123 @@
+
+CAVEAT-INPUT 
+ We recommend you use MQTT to interact with Ably when:
+
+ * Using devices which do not have a device-specific Ably SDK, such as Arduino platforms, C/C++ applications, and so on
+ * If bandwidth is limited and you want to keep network traffic to a minimum
+
+Read more in our "MQTT section":/protocols/mqtt.
+
+h3(#sse-eventsource). Ably SSE and EventSource API
+
+The Ably SSE and raw HTTP streaming API provides a way to get a realtime stream of events from Ably in circumstances where using a full Ably Realtime interface, or even an MQTT library, is impractical. 
+
+HTTP streaming allows for requests from clients to be held by a server, allowing it to push data to the client without further requests. This helps avoid the overhead involved in normal HTTP requests. Server-sent events (SSE) provide a thin layer on top of HTTP streaming. A common use of SSE is through the use of the EventSource API in all modern web browsers.
+
+Read more in our "SSE section":/protocols/sse.
+
+CAVEAT-INPUT 
+ Note that SSE is for subscribing to events only and you cannot publish to a channel. You will also have to handle certain operations in your client code, such as token renewal.
+
+h3(#pusher-pubnub). Pusher and PubNub protocols
+
+Ably provides the flexibility to migrate from Pusher progressively or all at once. 
+
+Our protocol adapters take care of all the background translation, you simply change your API key. You can also use a Pusher client library via this adapter to work on some platforms for which no Ably SDK is available.
+
+Take a look at our "Pusher tutorial":/tutorials/pusher-adapter and "PubNub tutorial":/tutorials/pubnub-adapter for step by step guidance.
+
+h2(#asset-tracking). Ably Asset Tracking SDKs
+
+Asset Tracking provides a way to track the location of multiple assets in realtime. An asset is any object that moves geographically, and whose position needs to be monitored. The Ably Asset Tracking solution is powered by the Ably realtime network, the Ably Asset Tracking SDKs, and the Mapbox Navigation SDK with location enhancement.
+
+Read more in our "asset tracking section":/asset-tracking/.
+
+h2(#integrations). Integrations
+
+Integrations enable data to be sent from Ably channels to external services and for external services to send data to Ably channels. We've broken integrations into inbound and outbound.
+
+Inbound streaming options are where an external service streams data to Ably, enabling you to publish data from an external web service into an Ably channel by registering a new webhook endpoint in Ably and configuring it within an external service. This allows you to easily connect external services to Ably.
```

Comment:
I'm not sure we should say "streams" here. Streaming is a very specific thing (such as streaming sound or video). For example, an inbound webhook firing on an external event such as inbound SMS or voice call would not be an example of streaming.